<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TEST MOTOR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>221</width>
       <height>176</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Init State :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Moving :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_MOVING">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_CURRENT">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Encoder :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_INIT_STATE">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_ENCODER">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="BTN_MOTOR_FIND_HOME">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Find Home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_MOTOR_SENSOR">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>80</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sensor :</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>221</width>
       <height>176</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_CURRENT_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_34">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Encoder :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_ENCODER_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_33">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Moving :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_MOVING_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LE_MOTOR_INIT_STATE_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_31">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_32">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Init State :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="BTN_MOTOR_FIND_HOME_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>320</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Find Home</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_MOTOR_CONNECTION">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>50</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connection :</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>271</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TEST LOADCELL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>231</width>
       <height>140</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QLineEdit" name="LE_LOADCELL_CALIB">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LE_LOADCELL_CONNECTION">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Error :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LE_LOADCELL_ERROR">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Connection :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Value :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Calib :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="LE_LOADCELL_VALUE">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="BTN_SET_CALIB">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set Calib</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BTN_LOADCELL_TARE">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tare</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_LOADCELL_SETTING_VALUE">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="text">
      <string>200</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>302</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_30">
       <property name="font">
        <font>
         <family>Lucida Console</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Board Connection :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_BOARD_CONNECTION">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lucida Console</family>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>---------------------------------------------------</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>---------------------------------------------------</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
